.circleci/config.yml

```diff
@@ -16,6 +16,10 @@ workflows:
           filters:
             branches:
               only: [main]
+      - update-lockfile:
```

Comment:
this should not run on the main branch instead it should run only on the release PR's (`release-please-*`)

Reply:
It doesn't matter what branch it runs on. Only that it runs after the release-please job. The job will checkout the correct branch, make changes in that branch and push the changes upstream.